benchmarking/collect_LIBSVM.sh

```diff
@@ -18,9 +18,8 @@ mkdir -p "${DEST_DIR}" || exit 1
 
```

Comment:
This bash script also needs a copyright header.  See collect_mittelmann_benchmark.sh for an example.

Reply:
Done.